recipes/pyee/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
We should make a PR to make sure this file is included in the pypi sdist, but this does not have to block merging the recipe.